### ISSUE TITLE
NXlog input created with restAPI mismatch for multiline and rename_check (sidecar=panic panic: interface conversion)

### ISSUE THREAD

**Issue by perbly** [Graylog2/graylog2-server#2260]
### Problem description
If a collector NXlog input is created with restAPI it could cause the collector-sidecar to panic

### Steps to reproduce the problem

1. I created a collector NXlog input using the restapi (http://graylog:12900/plugins/org.graylog.plugins.collector/configurations/572bb7efff52ed9fa0939ab4/inputs)

Body looks like this:
```
{
    "backend":  "nxlog",
    "properties":  {
                       **"rename_check":  "False",**
                       **"multiline":  "False",**
                       "save_position":  "True",
                       "recursive":  "True",
                       "poll_interval":  "1",
                       "read_last":  "True",
                       "multiline_start":  "/^-./",
                       "path":  "C:\\\\inetpub\\\\logs\\\\LogFiles\\\\W3SVC1\\\\u_ex*"
                   },
    "name":  "iis_w3svc1",
    "type":  "file",
    "forward_to":  "572bb8a9ff52ed9fa0939b79"
}
```

2. If i query the input this is the output:
```
{
    "backend":  "nxlog",
    "type":  "file",
    "name":  "iis_w3svc1",
    "properties":  {
                       **"rename_check":  "False",**
                       **"multiline":  "False",**
                       "save_position":  "True",
                       "recursive":  "True",
                       "poll_interval":  "1",
                       "read_last":  "True",
                       "multiline_start":  "/^-./",
                       "path":  "C:\\\\inetpub\\\\logs\\\\LogFiles\\\\W3SVC1\\\\u_ex*"
                   },
    "input_id":  "573d4f5eff52ed2428ea5fd1",
    "forward_to":  "572bb8a9ff52ed9fa0939b79"
}
```
3. If i now try to start the sidecar using: ".\graylog-collector-sidecar.exe -c .\collector_sidecar.yml" the sidecar crashes with this output:
```
time="2016-05-19T07:25:25+02:00" level=info msg="Using collector-id: 105699e3-caf7-4243-b837-8cd9cd3fc8a1"
time="2016-05-19T07:25:25+02:00" level=info msg="Fetching configurations tagged by: [windows iis]"
time="2016-05-19T07:25:25+02:00" level=info msg="Starting collector supervisor"
time="2016-05-19T07:25:25+02:00" level=info msg="[nxlog] Starting"
panic: interface conversion: interface is string, not bool

goroutine 7 [running]:
panic(0x88e780, 0xc0820ca500)
        /usr/local/Cellar/go/1.6.2/libexec/src/runtime/panic.go:481 +0x3f4
github.com/Graylog2/collector-sidecar/backends/nxlog.(*NxConfig).Add(0xc082158000, 0xc0820e4860, 0xa, 0xc0820ec580, 0x18
, 0x7c78e0, 0xc0820ea450)
        /Users/marius/Repositories/torch/go/src/github.com/Graylog2/collector-sidecar/backends/nxlog/configuration.go:13
6 +0xf83
github.com/Graylog2/collector-sidecar/backends/nxlog.(*NxConfig).RenderOnChange(0xc0820bc000, 0xc0820f4420, 0x2, 0x4, 0x
c082152240, 0x2, 0x4, 0xc082154000, 0x2, 0x4, ...)
        /Users/marius/Repositories/torch/go/src/github.com/Graylog2/collector-sidecar/backends/nxlog/render.go:310 +0x3e
8
github.com/Graylog2/collector-sidecar/services.checkForUpdateAndRestart(0xc082063140)
        /Users/marius/Repositories/torch/go/src/github.com/Graylog2/collector-sidecar/services/periodicals.go:59 +0x36d
github.com/Graylog2/collector-sidecar/services.StartPeriodicals.func2(0xc082063140)
        /Users/marius/Repositories/torch/go/src/github.com/Graylog2/collector-sidecar/services/periodicals.go:38 +0x28
created by github.com/Graylog2/collector-sidecar/services.StartPeriodicals
        /Users/marius/Repositories/torch/go/src/github.com/Graylog2/collector-sidecar/services/periodicals.go:40 +0x5e
```
4. Looking in the graylog gui for collectors i see that the input is created with "rename check" and "enable multiline" enabled. If i uncheck these the sidecar starts.

Very strange behaviour since the restAPI get methid returns the correct information but somehow the sidecar doesn´t look on the same place. 

### Environment

* Sidecar version: 0.0.7 (tried different)
* Graylog Version: 2.0.1
* Elasticsearch Version: 2.3.1
* MongoDB Version:
* Operating System: Win2012r2
* Browser version: Chrome 50

**Comment by perbly** [comment]
FYI the sidecar works if i untick the options in the gui and restart the sidecar.
Love the sidecar project btw.
 /Per

**Comment by mariussturm** [comment]
Hi,
thanks for the feedback! I think the problem is that you provide boolean values as strings. So in your API request you do e.g.: `"rename_check":  "False",` which should be `"rename_check":  false,`. Currently we don't have a strict type check there because we use the properties hash for all kind of collector backends. However the Sidecar should print an error in this situation and not just crash. I will take a look.

**Comment by mariussturm** [comment]
Should be fixed through c20da002b564d2bd034d169578b485c9df773279
